x-pack/plugins/ml/public/application/components/job_selector/job_selector.tsx

```diff
@@ -79,6 +81,9 @@ export interface JobSelectionMaps {
 
 export function JobSelector({ dateFormatTz, singleSelection, timeseriesOnly }: JobSelectorProps) {
   const [globalState, setGlobalState] = useUrlState('_g');
+  const [applyTimeRangeConfig, setApplyTimeRangeConfig] = useStorage<ApplyTimeRangeConfig>(
+    ML_APPLY_TIME_RANGE_CONFIG
+  );
```

Comment:
nit: you might want to provide the initial value as a second argument

Reply:
Updated here [`82b3c33` (#97407)](https://github.com/elastic/kibana/pull/97407/commits/82b3c330d69d78c0386ca7164db80736b36ffe0b)